_posts/2015-03-30-geomesa-feature-level-visibility.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Feature Level Visibility and Security
+author: jake
+layout: tutorial
+---
+
+{% include tutorial-header.html %}
+
+<!-- add some style to fix the xml formatting color -->
+<style>
+code.xml { color:#93a1a1 }
+</style>
+
+## Introduction
+
+Feature visibility is a new and important feature recently supported by GeoMesa. Accumulo has support for cell-level visibility built-in, and in recent versions of GeoMesa,
+support was added to utilize these visibility markings and enhance data security.
+<!--more-->
+From the Accumulo userguide:
+
+> Accumulo extends the BigTable data model to implement a security mechanism known as cell-level security. Every key-value pair has its own security label, stored under the column visibility element of the key, which is used to determine whether a given user meets the security requirements to read the value. This enables data of various security levels to be stored within the same row, and users of varying degrees of access to query the same table, while preserving data confidentiality.
+
+In this tutorial, you'll be guided through ingesting data with varying levels of visibility and querying that data as different users through GeoServer.
+
+## Prerequistes
+
+If you haven't already [read through the Quickstart tutorial](http://www.geomesa.org/2014/05/28/geomesa-quickstart/) and
+gone through the initial setup of GeoMesa, read through that first. We'll be using a customized version of the data generated by the GeoMesa Quickstart project.
+
```

Comment:
Add something about 'user should understand accumulo visibilities' and link to the accumulo user guide on them.